sharktank/tests/models/llama/benchmark_amdgpu_test.py

```diff
@@ -571,6 +600,7 @@ def testBenchmark70B_f16_TP8_Non_Decomposed_Input_Len_128(self):
             args=self.compile_args,
         )
         # benchmark prefill
+        logger.info("IREE Benchmark Prefill...")
```

Comment:
nit: when comments and logs are saying the same thing, you can remove the comments